tourneydefs.py

```diff
@@ -85,6 +85,9 @@ class Tournament(BaseModel):
     game_history: List[Game] = []
     mapping: Dict = {}
     version : str = "0.3"
+    pts_on_win: int = 0
+    pts_on_tie: int = 0
+    pts_on_loss: int = 0
```

Comment:
This feels like it should be a tournament configuration option - my rec here is have a dict property so we can auto-export/import any new config options that get added.

`config: Dict = {     pts_on_win: 1
    pts_on_tie: 0
    pts_on_loss: 0 }`

Also feel free to write getter/setter for this to avoid keyerrors and ease refactoring